backends/ris/decoder.go

```diff
@@ -205,14 +205,11 @@ func mapRecord(r Record, p *models.Publication) {
 				p.Author = nil
 			}
 			for _, val := range v {
-				nameParts := reSplit.Split(val, -1)
-				lastName := nameParts[0]
-				firstName := "[missing]" // TODO
-				if len(nameParts) > 1 {
-					firstName = nameParts[1]
-				}
-				c := models.ContributorFromFirstLastName(firstName, lastName)
-				p.Author = append(p.Author, c)
+				p.Author = append(p.Author, extractContributor(val))
+			}
+		case "BE", "ED":
+			for _, val := range v {
+				p.Editor = append(p.Editor, extractContributor(val))
 			}
```

Comment:
Currently if both are present, both are imported, as opposed to authors where code `AF` is preferred over `AU`. Can be improved if necessary obviously.

Reply:
Thanks! I added some more information here, please let me know if this clarifies the question: https://github.com/ugent-library/biblio-backoffice/issues/1625#issuecomment-2256129725